verticapy/sdk/vertica/udf/load.py

```diff
@@ -39,60 +39,59 @@ def import_lib_udf(
     Parameters
     ----------
     udf_list: list
-        List of tuples that includes the different functions.
-
-        **function**     :
-                            [function]  Python   Function.
-
-        **arg_types**    :
-                            [dict/list] List or dictionary
-                            of  the function input  types.
-
-                            Example: {"input1": int,
-                            "input2": float}  or
-                            [int, float]
-
-        **return_type**  :
-                            [type/dict] Function output type.
-                            In the case of many  outputs, it
-                            must be a dictionary including
-                            all the outputs types and names.
-
-                            Example: {"result1": int,
-                            "result2": float}
-
-        **parameters**   :
-                            [dict] Dictionary of the function
-                            input optional parameters.
-
-                            Example: {"param1": int,
-                            "param2": str}
-
-        **new_name**     :
-                            [str] New   function   name  when
-                            installed in Vertica.
+        List of tuples that includes the different
+        functions.
+
+         - function: function
```

Comment:
I am not very sure about this use.
When using " - " for list bold with ":", there should not be any text in that line after ":".